api/app/src/shared/types.ts

```diff
@@ -1,2 +1,7 @@
 // https://stackoverflow.com/a/48244432/2099911
 export type AtLeastOne<T, U = { [K in keyof T]: Pick<T, K> }> = Partial<T> & U[keyof U];
+
+export function stringToBoolean(value?: string): boolean | undefined {
+  if (!value) return undefined;
+  return value === "true";
```

Comment:
nit: could use `value.toLower()` before testing so that `True` and `TRUE` are also valid